### ISSUE TITLE
Map crashes

### ISSUE THREAD

**Issue by DulongSang** [CMPUT301W20T29-H03/ZoomWroom#145]
The app crashes after I login as a rider or driver, in the map activity, the app got frozen and I was not able to click anything. I do not know if anyone has the same problem.

**Comment by sharyar** [comment]
@DulongSang , can you add the log cat errors here so we can find the issue?

**Comment by DulongSang** [comment]
Also, there are some problems in the driver and rider sign up activities, I could log in but the UI was broken.

**Comment by DulongSang** [comment]
> @DulongSang , can you add the log cat errors here so we can find the issue?

I cannot find the issue because it got stuck, unlike the app actually crash as before.
I got a message from Android that said: "Zoom Wroom isn't responding, [Close app] [Wait]".
It did not seem like unhandled exceptions, it was like an infinite loop or something like this because my memory and CPU usage went extremely high.
I did not find any weird message in logcat or run console
